lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -162,5 +163,12 @@ sub parent_group_overview {
     $self->group_overview('JobGroupParents', 'main/parent_group_overview');
 }
 
+sub changelog {
+    my ($self) = @_;
+    my $changelog = decode('utf8', `rpm -q --changelog openQA`);
```

Comment:
this needs error handling. openQA does not need to be installed from rpm - i.e. in tests